src/test/java/org/jenkinsci/plugins/envinject/EnvInjectVarListTest.java

```diff
@@ -16,44 +16,46 @@
  * @author Oleg Nenashev
  */
 public class EnvInjectVarListTest {
-    
-    @Rule 
+
+    @Rule
     public JenkinsRule j = new JenkinsRule();
 
-    @Test 
+    @Test
     @Issue("JENKINS-44263")
     public void envVarsShouldBeCachedProperlyAfterReload() throws Exception {
         final FreeStyleProject p = j.jenkins.createProject(FreeStyleProject.class, "project");
         p.getBuildersList().add(new EnvInjectBuilder(null, "TEXT_VAR=tvalue"));
         final FreeStyleBuild build = j.buildAndAssertSuccess(p);
-        
+
         // Check vars before the reload
         {
             EnvInjectVarList varList = getVarListOrFail(build);
             Map<String, String> envMap = varList.getEnvMap();
             Assert.assertNotNull("EnvInject vars list is null for the run before reload", envMap);
             Assert.assertTrue("TEXT_VAR is not present in the list", envMap.containsKey("TEXT_VAR"));
         }
-        
+
         // Reload and check vars
         // build.reload() does not assign parents for RunAction2, hence we apply a workaround
         p.doReload();
         final Run<?, ?> reloadedBuild = p.getBuildByNumber(build.getNumber());
         {
             EnvInjectVarList varList = getVarListOrFail(reloadedBuild);
+            Assert.assertNotNull("EnvInject varList is null for the run after the reload", varList);
+
             Map<String, String> envMap = varList.getEnvMap();
-            Assert.assertNotNull("EnvInject vars list is null for the run after the reload", envMap);
+            Assert.assertNotNull("EnvInject envMap from varList is null for the run after the reload", envMap);
             Assert.assertTrue("TEXT_VAR is not present in the list", envMap.containsKey("TEXT_VAR"));
             Assert.assertEquals("TEXT_VAR has wrong value", "tvalue", envMap.get("TEXT_VAR"));
```

Comment:
The commented problem about test that broke between 2.264 and 2.269 is here.

Reply:
most likely xstream changes https://www.jenkins.io/changelog/#v2.266